coalib/bears/Bear.py

```diff
@@ -301,7 +302,12 @@ def __init__(self,
         self.section = section
         self.message_queue = message_queue
         self.timeout = timeout
-        self.debugger = debugger
+        self.debugger = (False if 'debug_bears' not in section or (
```

Comment:
@jayvdb I didn't get, you want me to make a different private method which do the same work?

>You will use it again for profile

I don't think so as `debug_bears` accept different argument then `profile`, If I try to mix both it will create a extra code

Reply:
Also IMO creating a helper will not help as both `debug_bears` and `profile` take different argument and accordingly logic change in both case.

Reply:
the helper would take an argument.

If necessary create two helpers.

At the moment these are gibberish logic which I am assuming is similar because it isnt python code which is readable and intuitive.